config.json

```diff
@@ -216,6 +216,16 @@
         "pattern_matching",
         "stacks"
       ]
+    },
+    {
+      "slug": "isogram",
```

Comment:
Recommend moving this to after armstrong-numbers

Reply:
Moved.